tailwindcss-config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  purge: [],
+  theme: {
+    extend: {},
+  },
+  variants: {},
+  plugins: [],
+}
```

Comment:
Why those are empty ?

Reply:
i didnt used it maybe i will delete it and for the index.css it contains the tailwind i cant remove it , it is necessary ...so now should I apply the i18next ?

Reply:
yeah please apply the i18next 
and for the index.css leave it now and we will talk about later